src/test/java/eu/enmeshed/EnmeshedOnboardingServiceTest.java

```diff
@@ -56,15 +56,14 @@ public class EnmeshedOnboardingServiceTest {
       List.of(GivenName.class, Surname.class);
   private static final List<Class<? extends AttributeValue>> OPTIONAL_ATTRIBUTES =
       List.of(EMailAddress.class);
-  private final List<Class<? extends RequestItem>> CREATE_ATTRIBUTES =
-      List.of(CreateAttributeRequestItem.class);
-
   private static final IdentityInfo TEST_IDENTITY_INFO =
       IdentityInfo.builder()
           .address("da88dd1b2b820360d4155162e657f84ea1394076faa1ce2909d8338811cb308d")
           .publicKey("dbb5d8fd21caf827fdc128d73e783478d6677a9afc50120db56217726125425f")
           .realm("4354b5ae54bab15544f852d7bc1b76bbd6d71a03b5e7ad876916cda3a602aaf9")
           .build();
+  private final List<Class<? extends RequestItem>> CREATE_ATTRIBUTES =
```

Comment:
Would RequestItemDerivation be suitable as well as the type?

Reply:
the compiler doesn't like that

Reply:
don't ask me why.

Reply:
Alright, the compiler wants what the compiler wants 👸